code/__DEFINES/traits.dm

```diff
@@ -215,6 +215,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_DISK_VERIFIER "disk-verifier"
 #define TRAIT_NOMOBSWAP "no-mob-swap"
 #define TRAIT_XRAY_VISION "xray_vision"
+#define TRAIT_WEB_WEAVER "web_weaver"
```

Comment:
Document.

Reply:
should be done, if I correctly understood what you meant by "document"